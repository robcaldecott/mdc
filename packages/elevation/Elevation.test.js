import React from "react";
import { Elevation } from ".";
import renderer from "react-test-renderer";

it("renders", () => {
  expect(
    renderer.create(
      <Elevation z={4} transition>
        Elevation
      </Elevation>
    )
  ).toMatchSnapshot();
});
