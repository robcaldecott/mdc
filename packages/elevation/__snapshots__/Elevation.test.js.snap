// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="mdc-elevation--z4 mdc-elevation-transition"
>
  Elevation
</div>
`;
